cmd/util.go

```diff
@@ -290,13 +295,23 @@ Alternatively, you can specify the version with --version
 `
 
 func getKubeVersion() (*KubeVersion, error) {
+	kubeConfig, err := rest.InClusterConfig()
```

Comment:
The code performs multiple operations that can return errors, but it doesn't handle these errors adequately. Consider adding appropriate error handling to provide better feedback to users when issues occur.